spec/Modules.tex

```diff
@@ -9,9 +9,10 @@
 Module definitions are described in~\rsec{Module_Definitions}.  The
 relation between files and modules is described
 in~\rsec{Implicit_Modules}.  Nested modules are described
-in~\rsec{Nested_Modules}.  Module uses and explicit naming of symbols
-are described in~\rsec{Using_Modules}.  The execution of a program and
-module initialization is described in~\rsec{Program_Execution}.
+in~\rsec{Nested_Modules}.  The visibility of symbols within a module
+in relation to other modules is described in~\rsec{Visibility}.  The
+execution of a program and module initialization is described
```

Comment:
I'd like it to say "The visibility of a module's symbols by users of the module is described in~\rsec{Visibility}."